app/javascript/gobierto_data/webapp/components/commons/Description.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <Dropdown @is-content-visible="showDescription = !showDescription">
+      <template v-slot:trigger>
+        <h2 class="gobierto-data-tabs-section-title">
+          <Caret :rotate="showDescription" />
+          {{ labelDescription }}
+        </h2>
+      </template>
+      <div class="gobierto-data-description">
+        <template
+          v-for="(row, i) in numberOfRows"
+        >
+          <div
+            v-for="{ column, type } in arrayOfObjectsColumns.slice(i * itemsPerRow, (i + 1) * itemsPerRow)"
```

Comment:
What did this do?

Reply:
At first it made sense, now that we build the columns with CSS it doesn't make sense. Refactor